dependencies/CMakeLists.txt

```diff
@@ -10,6 +10,12 @@ include(ExternalProject)
 include(CMakeParseArguments)
 include(CMakeDependentOption)
 
+set(
+    SIMBODY_EXTRA_CMAKE_ARGS ""
+    CACHE STRING
+    "extra args to forward when configuring Simbody with CMake. The format must be: VARNAME1:TYPE1=VALUE1;VARNAME2:TYPE2=VALUE2"
```

Comment:
(optional) Point to a list of possible VARNAMEs or provide one example (e.g., the arguments you are using) to hopefully make this more accessible to users?

Reply:
I've clarified the documentation slightly in 1fce59e